lib/Builder.php

```diff
@@ -198,4 +196,30 @@ private function render(string $directory, string $targetDirectory) : void
             new PostBuildRenderEvent($this, $directory, $targetDirectory)
         );
     }
+
+    private function loadCachedMetas(string $targetDirectory) : void
```

Comment:
Can this be refactored out to a separate class that deals with the reading and writing of metas?